src/Components/ProfileSection/ProfileSection.tsx

```diff
@@ -1,14 +1,23 @@
 import { Profile } from '@/Assets';
 import styled from 'styled-components';
+import { SettingButton } from './SettingButton';
+
+interface ProfileSectionProps {
+  nickname: string;
+  email: string;
+}
```

Comment:
```suggestion
export interface ProfileSectionProps {
  /** 사용자 이름 */
  nickname: string;
  
  /** 사용자 이메일 주소 */
  email: string;
}
```

![image](https://github.com/Ludo-SMP/ludo-frontend/assets/72962900/c0cfebfa-b001-47e4-8062-98afbf9f32d5)

export를 하면 문서화 등에 도움이 될 것 같아요!